src/tests/ftest/launch.py

```diff
@@ -2263,6 +2266,41 @@ def indent_text(indent, text):
     return " " * indent + str(text)
 
 
+def reset_server_storage(test_file, args):
+    """Reset the server storage for the hosts that ran servers in the test.
+
+    This is a workaround to enable binding devices back to nvme or vfio-pci after they are unbound
+    from vfio-pci to nvme.  This should resolve the "NVMe not found" error seen when attempting to
+    start daos engines in the test.
+
+    Args:
+        test_file (dict): a dictionary of the test script/yaml file
+        args (argparse.Namespace): command line arguments for this program
+
+    Returns:
+        int: status code: 0 = success, 512 = failure
+
+    """
+    server_hosts = get_hosts_from_yaml(test_file["yaml"], args, YAML_KEYS["test_servers"])
+    print("-" * 80)
+    if server_hosts:
+        commands = [
+            "if lspci | grep -i nvme",
+            "then daos_server storage prepare -n --reset && " +
+            "sudo rmmod vfio_pci && sudo modprobe vfio_pci",
+            "fi"]
+        print(
+            "Resetting server storage on {} after running '{}'".format(
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string